src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -43,6 +43,11 @@ public static int Run(string[] args)
             CommandOption versionSuffixOption = app.Option(
                $"--version-suffix <{LocalizableStrings.VersionSuffixOption}>", LocalizableStrings.VersionSuffixOptionDescription,
                 CommandOptionType.SingleValue);
+
+            CommandOption profileProjOption = app.Option(
+               $"--profile <{LocalizableStrings.ProfileProjOption}>", LocalizableStrings.ProfileProjOptionDescription,
```

Comment:
It feels strange that `profile` maps to a property named `FilterProjFile` in MSBuild.  It would make sense, IMO, if these two properties used the same terms.